### ISSUE TITLE
LuaU Support

### ISSUE THREAD

**Issue by Trimblehubxyz** [zeusssz/hercules-obfuscator#35]
It would be nice for roblox LUA support man

**Comment by zeusssz** [comment]
yes, we are working on that. itll come around v1.7-v1.8, when hercules becomes less static

**Comment by ImInTheICU** [comment]
I'd hope to also see LuaU support.

**Comment by Ostensibly-1** [comment]
> It would be nice for roblox LUA support man

it already supports roblox lua, it just doesnt have those fancy types, type checking ,compound statements, and continue statement thingies that u luau duds are using
